pytest/tests/mocknet/mirror.py

```diff
@@ -219,13 +220,55 @@ def reset_cmd(args, traffic_generator, nodes):
         )
         if sys.stdin.readline().strip() != 'yes':
             sys.exit()
+    if args.backup_id is None:
+        backups = nodes[0].neard_runner_ls_backups()
+        backups_msg = 'ID |  Time  | Description\n'
+        if 'start' not in backups:
+            backups_msg += 'start | None | initial test state after state root computation\n'
+        for backup_id, backup_data in backups.items():
+            backups_msg += f'{backup_id} | {backup_data.get("time")} | {backup_data.get("description")}\n'
+
+        print(f'Backups as reported by {nodes[0].name()}):\n\n{backups_msg}')
+        print('please enter a backup ID here:')
+        args.backup_id = sys.stdin.readline().strip()
+        if args.backup_id != 'start' and args.backup_id not in backups:
+            print(
+                f'Given backup ID ({args.backup_id}) was not in the list given')
+            sys.exit()
```

Comment:
This check should happen both for backup_id provided in args and in readline (so outside this if)

Reply:
mm well the reason i left it out when passing the backup ID on the command line was that when you dont pass it on the command line, we want to call `ls-backups` to get a list of backups to show the user. but when you do pass it on the command line, we can just skip that extra round trip and send it anyway, and `neard_runner.py` will just check that it doesnt exist. so you get the same behavior where `neard_runner.py` tells you it doesnt exist, but with only one RPC call